### PR TITLE
Added some test for priority

### PR DESCRIPTION
**Pull request by jbeaudoin11**
#733

### CODE REVIEW

File: test/test_queue.js [hunk 1]
```diff
@@ -474,6 +475,89 @@ describe('Queue', function () {
         }, done);
     });
 
+    function priorityTestSetup(numJobsPerPriority, nbWorkers, done) {
+      var jobsPrioirtyInCompletedOrder = []
+      queue.process(nbWorkers, function(job, jobDone){
```
Comment (manast) [inline test/test_queue.js]: typo in ```jobsPrioirtyInCompletedOrder```

File: test/test_queue.js [hunk 2]
```diff
@@ -474,6 +475,89 @@ describe('Queue', function () {
...
+      // When jobs are all completed
+      var intervalId = setInterval(function() {
+        if(jobsPrioirtyInCompletedOrder.length === numJobsPerPriority * 2) {
+          clearInterval(intervalId);
+
+          var trueJobsPrioirtyInCompletedOrder = []; // [p1, p1, p1, ..., p2, p2, p2]
+          for(var p=1; p<3; p++) {
+            for(var i=0; i<numJobsPerPriority; i++) {
+              trueJobsPrioirtyInCompletedOrder.push(p)
+            }
```
Comment (manast) [inline test/test_queue.js]: typo "prioirty"

Comment (jbeaudoin11) [reply test/test_queue.js]: Okay ty

**Comment by manast** [comment]
not sure this PR is relevant anymore, if there is interest, if after fixing the conflicts it becomes green I can merge it though.

**Pull request closed without merging**